packages/google_sign_in/lib/google_sign_in.dart

```diff
@@ -201,8 +205,11 @@ class GoogleSignIn {
   /// a Future which resolves to the same user instance.
   ///
   /// Re-authentication can be triggered only after [signOut] or [disconnect].
-  Future<GoogleSignInAccount> signInSilently() =>
-      _addMethodCall('signInSilently');
+  Future<GoogleSignInAccount> signInSilently() {
+    return _addMethodCall('signInSilently').catchError((dynamic _) {
+      // ignore, we promised to be silent.
```

Comment:
I don't think we want to swallow *all* errors here.  The problem was that the OEM code was throwing too aggressively.  Once that's fixed, I think we want to propagate normal exceptions here.